django_mongodb/compiler.py

```diff
@@ -212,8 +231,20 @@ def _get_ordering(self):
             field_ordering.append((opts.get_field(name), ascending))
         return field_ordering
 
+    @cached_property
+    def collection_name(self):
+        return self.query.get_meta().db_table
+
     def get_collection(self):
-        return self.connection.get_collection(self.query.get_meta().db_table)
+        return self.connection.get_collection(self.collection_name)
+
+    def get_lookup_pipeline(self):
```

Comment:
Should this method be private?

Reply:
I suppose it would be consistent with `_get_ordering()`. On the other hand, I'm not sure I would have underscore prefixed that method (that code came from the original fork). When I look at Django's SQL compiler, underscore methods are mainly used for utility methods used more than once in other methods.

This class doesn't have any methods Django developers call themselves, so whether or not to underscore is mainly a decision about how we want to organize things.